scripts/count-median.py

```diff
@@ -34,8 +36,18 @@ def get_parser():
     based on the k-mer counts in the given k-mer counting table.  Can be used
     to estimate expression levels (mRNAseq) or coverage (genomic/metagenomic).
 
-    The output file contains sequence id, median, average, stddev, and seq
-    length.
+    The output file contains sequence id, median, average, stddev, and
+    seq length; fields are separated by spaces.  For khmer 1.x,
+    sequence names are split at the first space, which means that some
+    sequence formats (e.g. paired FASTQ in Casava 1.8 format) will yield
+    uninformative names.  Use :option:`--csv` to fix this behavior.
+
+    :option:`--csv` will output a comma-separated value file with column
+    headers, and the entire sequence name will be kept.
```

Comment:
I think the redundancy is not harmful, in this case, and may be helpful to people who only skim docs.